runtime/compiler/control/CompilationThread.hpp

```diff
@@ -49,6 +49,11 @@
 #include "env/J2IThunk.hpp"
 #include "env/PersistentCollections.hpp"
 
+extern "C" {
+   int32_t getCount(J9ROMMethod *romMethod, TR::Options *optionsJIT, TR::Options *optionsAOT);
```

Comment:
I would prefer this not to be included in the hpp file. If you refactor the code like I suggested below and put the refactored method in CompilationThread.cpp, then you can eliminate these lines